tests/unit/App.spec.ts

```diff
@@ -159,17 +171,26 @@ describe('App component', () => {
           }
       })))
 
+    // create a Local Vue and install router on it
     const localVue = createLocalVue()
     localVue.use(VueRouter)
     const router = mockRouter.mock()
     router.push({ name: 'define-company', query: { nrNumber: 'NR 1234567' } })
-    wrapper = shallowMount(App, { sync: false, localVue, store, router, vuetify, stubs: { Affix: true } })
+    wrapper = shallowMount(App, {
+      sync: false,
+      localVue,
+      store,
+      router,
+      vuetify,
+      stubs: { Affix: true, ConfirmDialog }
```

Comment:
Note the "ConfirmDialog" component here. This is so that it is correctly mounted/rendered. All other components are stubbed because this is a shallow mount.